modules/Makefile.kubernetes

```diff
@@ -167,7 +167,12 @@ kubernetes\:run-job:
 ## Show job logs
 kubernetes\:job-logs:
 	@echo -e "INFO: Job logs for $(KUBERNETES_APP) on cluster $(call yellow,$(CLUSTER_NAMESPACE))..."
-	@$(KUBECTL_CMD) logs -f $(shell $(KUBECTL_CMD) get pods -a -o name | grep $(KUBERNETES_APP))
+	@while [[ $$JOB_STATUS != "Running" && $$JOB_STATUS != "Completed" && $$JOB_STATUS != "Failed" ]]; do \
```

Comment:
nit: have it output a message indicating its waiting for job to run, then output `Current job status: x`, and finally a newline or separator between the final status and the begin of the output. It will make it easier for everyone to follow what it is doing